### PR TITLE
New branch with react code

### PR DESCRIPTION
**Pull request by shantanuFyndGit**

### CODE REVIEW

File: constants.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	{
+		name: "Refine",
+		title: "Refine Output",
+		type: "boolean",
+		default: true,
+		identifier: "r",
+	},
+];
+
+export enum msgTypes {
```
Comment (sagarika-fynd) [inline constants.ts]: rename to EVENTS

File: plugin/index.ts [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+figma.showUI(__html__, {
+	title: "Erase.bg",
+	height: 400,
+	width: 248,
+	themeColors: true,
+});
+
+const rectangles: RectangleNode[] = [];
+
+if (figma.command === "how-it-works-command") {
```
Comment (sagarika-fynd) [inline plugin/index.ts]: move to enum called commands

File: plugin/index.ts [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+	var savedToken;
+	if (msg.type === msgTypes.INITIAL_CALL) {
+		const body = {
+			type: msgTypes.CREATE_FORM,
+			optionsArray: eraseBgOptions,
+			savedFormValue: "",
+		};
+
+		try {
+			savedToken = await figma.clientStorage.getAsync("persistedToken");
```
Comment (sagarika-fynd) [inline plugin/index.ts]: move to constant files

File: ui/App.tsx
Comment (sagarika-fynd) [inline ui/App.tsx:1]: please decrease the size of this file

**Review by sagarika-fynd — CHANGES_REQUESTED** [review]

File: constants.ts (hunk 1 above)
Comment (shantanuFyndGit) [reply constants.ts]: resolved

File: plugin/index.ts (hunk 2 above)
Comment (shantanuFyndGit) [reply plugin/index.ts]: resolved

File: plugin/index.ts (hunk 3 above)
Comment (shantanuFyndGit) [reply plugin/index.ts]: resolved

**Pull request merged**